src/home/Home.tsx

```diff
@@ -4,15 +4,14 @@ import "./_home.scss";
 
 import React from "react";
 import {useSelector} from "react-redux";
-import {RouteComponentProps} from "@reach/router";
 
 import {authenticationProfileSelector} from "../authentication/redux/util/authenticationReduxUtils";
 
 export interface HomeOwnProps {
```

Comment:
We no longer need to export this, as we wont be using routeConfig
```suggestion
interface HomeProps {
```